### PR TITLE
initial setup upload

### PR DESCRIPTION
**Pull request by joshin-run**
index and stylesheet

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>StoryStart</title>
+		<link type="text/css" rel="stylesheet" href="stylesheet.css"/>
+    </head>
+    <body>
+        
+		<form action="mailto:someone@example.com" method="post" enctype="text/plain">
```
Comment (johnhoffman) [inline index.html]: fix inconsistent indentation

File: index.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    </head>
+    <body>
+        
+		<form action="mailto:someone@example.com" method="post" enctype="text/plain">
+            
+        <div class="div-h">
+            <h1>StoryStart</h1>
+        </div>
+        <div class="div-p">
+            <div><p>NAME YOUR STORY<br/>
```
Comment (johnhoffman) [inline index.html]: For readability, tags such as the <p> usually either 1) vertically line up with the corresponding closing tag </p> or 2) are on the same line as the closing tag. In this case, the former would be appropriate.

File: stylesheet.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
+body {
+    background-color: #fff;
+}
+/*.div-mid {
```
Comment (johnhoffman) [inline stylesheet.css]: typically, remove commented-out code before committing/merging

**Comment by johnhoffman** [comment]
Overall, looks good! Main thing to improve is HTML formatting. One idea could be to use a Sublime Text plugin to clean up formatting for you.

This looks like it could be good:
https://github.com/victorporof/Sublime-HTMLPrettify

You can install it using the Sublime Package Manager, and that's probably the easiest way. There's a little section in the README for how to do that, but for convenience, here it is:
- Ctrl+Shift+P or Cmd+Shift+P in Linux/Windows/OS X
- type install, select Package Control: Install Package
- type prettify, select HTML-CSS-JS Prettify

Once you clean up the formatting, I'll be happy to merge this for you. All you need to do is update the code on this branch (dev-form_boot), commit, and push. This pull request will auto-update with your latest changes.

**Comment by joshin-run** [comment]
Please review and merge

**Comment by johnhoffman** [comment]
Great work!

**Pull request merged**